probables/countminsketch/countminsketch.py

```diff
@@ -8,6 +8,9 @@
 import os
 from numbers import Number
 from struct import calcsize, pack, unpack
+from typing import Iterable
+
+import numpy as np
```

Comment:
this is a big dependency to add

Reply:
I know. I plan to perform and add benchmark demonstrating the efficiency so we can decide how much we gain using such big dependency.